src/rt/util/hash.d

```diff
@@ -95,15 +90,18 @@ size_t hashOf( const(void)* buf, size_t len, size_t seed )
     return hash;
 }
 
-// Check that hashOf works with CTFE
-@nogc nothrow pure unittest
```

Comment:
Yes, I was uncomfortably aware of that when I removed the attributes. When I wrote the n.g. post, I had forgotten that the point of attributes on unittests is to test the attribute inference of the functions/templates it calls. In this case, the attributes of hashOf are fully specified, so adding them to the unittest does nothing, and is inconvenient because of the printf.

`debug` offers an escape from purity not @safe. Perhaps it would be a reasonable enhancement to add that.

And I'm gratified that somebody reads my broadsides :-)